### DISCUSSION TITLE
Migration from Trinity to firefly does not work using migration tool

### DISCUSSION THREAD

**Discussion by gernotmuc** [iotaledger/legacy-migration-tool#149] [Help & Support] [answered]
Hi ! Does anybody know how to solve the 
![Bildschirmfoto vom 2024-12-29 13-34-49](https://github.com/user-attachments/assets/5b638278-7805-4a2f-ad8e-d533f229cced)
issue ?
The migration log contains:
errorMessage: cannot debit (base tokens: 1000959041) from 05204969@iota1pphx6hnmxqdqd2u4m59e7nvmcyulm3lfm58yex5gmud9qlt3v9crs9sah6m: not enough funds

**Comment by michalszot123** [comment]
You are find solution this issu? I have the same problem

**Reply by shicksreliant** [reply]
me too.  plenty of funds.  but errors out as "not enough funds" which makes me hypothesize it's somehow related to how many decimal places 1 IOTA is considered to have.

**Comment by gernotmuc** [comment, marked as answer]
Unfortunately not yet ...

**Reply by jasonchen1980** [reply]
qq

**Comment by wako68** [comment]
does anybody know how to solve the problem now? I have the same issu

**Comment by Andy0709wejobbr** [comment]
i get the same error too. any solution?

balance: 419000000 
errorMessage: cannot debit (base tokens: 419000000) from XXXXXXXXXXXXXXXXXXXXXXXX: not enough funds

**Comment by begonaalvarezd** [comment]
Hi, we have an issue with the migrator but we are working on it, we are very sorry for the troubles, will notify you when the issue is solved 🙏🏼

**Reply by Andy0709wejobbr** [reply]
i appretiate your fast response, thank you! -good luck!

**Reply by kirankn** [reply]
Would appreciate an ETA. Thanks for your efforts.

**Comment by begonaalvarezd** [comment]
The issue `errorMessage: cannot debit (base tokens: XXX) from YYY: not enough funds` has been mitigated, please try the migration again. We apologize for the troubles :pray:
